spec/stub_helper.rb

```diff
@@ -0,0 +1,134 @@
+def cross_streets
```

Comment:
I use it in my nokogiri seeding task to produce cross streets so that when users (eventually) view bike rack details, it will have that information. It will also attempt to add that info (eventually...) when a user adds a new bike rack. It's a pretty cool service!